Platform/MeteorlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -835,6 +802,36 @@ UpdateFspConfig (
   FspsConfig->SerialIoSpiMosiPinMux[0] = 0x14a48d;
   FspsConfig->DmiPowerGatingDis = 0x0;
 
+  MaxPcieRootPorts = GetPchMaxPciePortNum ();
+  for (Index = 0; Index < MaxPcieRootPorts; Index++) {
+    FspsConfig->PcieRpLtrMaxSnoopLatency[Index]          = 0x100F;
+    FspsConfig->PcieRpLtrMaxNoSnoopLatency[Index]        = 0x100F;
+    FspsConfig->PcieGen3EqPh3NoOfPresetOrCoeff[Index]    = 0x5;
+    FspsConfig->PcieGen3EqPh3PostCursor0List[Index]      = 0xc;
+    FspsConfig->PcieGen3EqPh3PostCursor2List[Index]      = 0xa;
+    FspsConfig->PcieGen3EqPh3PreCursor9List[Index]       = 0x0;
+    FspsConfig->PcieGen3EqPh3Preset4List[Index]          = 0x0;
+    FspsConfig->PcieGen3EqPh3Preset5List[Index]          = 0x8;
+    FspsConfig->PcieGen4EqPh3NoOfPresetOrCoeff[Index]    = 0x3;
+    FspsConfig->PcieGen4EqPh3PostCursor0List[Index]      = 0xc;
+    FspsConfig->PcieGen4EqPh3PostCursor2List[Index]      = 0xa;
+    FspsConfig->PcieGen5EqPh3NoOfPresetOrCoeff[Index]    = 0x4;
+    FspsConfig->PcieGen5EqPh3PostCursor0List[Index]      = 0xe;
+    FspsConfig->PcieGen5EqPh3PostCursor1List[Index]      = 0xa;
+    FspsConfig->PcieGen5EqPh3PostCursor2List[Index]      = 0xc;
+    FspsConfig->PcieGen5EqPh3PostCursor7List[Index]      = 0xa;
+    FspsConfig->PcieGen5EqPh3PreCursor9List[Index]       = 0x8;
+    FspsConfig->PcieRpUnsupportedRequestReport[Index]    = 0x1;
+    FspsConfig->PcieRpFatalErrorReport[Index]            = 0x1;
+    FspsConfig->PcieRpNoFatalErrorReport[Index]          = 0x1;
+    FspsConfig->PcieRpCorrectableErrorReport[Index]      = 0x1;
+    FspsConfig->PcieRpSnoopLatencyOverrideMode[Index]    = 0x2;
+    FspsConfig->PcieRpNonSnoopLatencyOverrideMode[Index] = 0x2;
+    FspsConfig->PcieRpPhysicalSlotNumber[Index]          = (UINT8)Index;
+  }
+  FspsConfig->PcieGen3EqPh3Preset5List[11]               = 0x9;
+  FspsConfig->PcieGen4EqPh3NoOfPresetOrCoeff[11]         = 0x5;
```

Comment:
It looks this change is not related with Coverity issue as mentioned in the commit message. maybe split this PR into 2 PR?

Reply:
This is the portion i moved from line 554. No new value was assigned to UPD.

Reply:
patch updated